sktime/forecasting/fbprophet.py

```diff
@@ -142,7 +151,7 @@ def __init__(
         uncertainty_samples=1000,
         stan_backend=None,
         verbose=0,
-        interval_width=0,
+        interval_width=0.80,
```

Comment:
Synced this [default parameter ](https://github.com/facebook/prophet/blob/b7d5ec448177b7ac2abf25cc3259cb55e4c44069/python/prophet/forecaster.py#L93)value with Prophet.